container/root/s6-setuidgid

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+current_user=$(whoami)
```

Comment:
this seems like something we might want to maintain in the base, no? 

no immediate action, but let's get this out, then uplevel to docker-base after

Reply:
Yes, I was actually debating that but was not sure if we were ready to make changes to the base yet.